Sources/Avatar.swift

```diff
@@ -40,10 +40,9 @@ public struct Avatar {
     
     public func image(highlighted: Bool) -> UIImage {
         
-        switch highlighted {
```

Comment:
I'm one of those developers that doesn't like if statements for no good reason :P I try to avoid them at all costs!